concent_api/conductor/models.py

```diff
@@ -34,6 +34,9 @@ class VerificationRequest(Model):
     # Deadline on additional verification for related Subtask.
     verification_deadline = DateTimeField()
 
+    # Indicates when Conductor has received the request.
+    created_at = DateTimeField(default=timezone.now)
```

Comment:
We should use `auto_now_add=True`. You can change this on existing fields which use `timezone.now`.

Reply:
I read yesterday that if we use `auto_now_add` option then this won't be visible in administration panel - it was some bug in django. Maybe now this bug is already fixed because this was information from before year or two.

Done

Reply:
I have discovered something strange. If you want to add `default` to field definition you can't do this when you have `auto_now` or `auto_now_add` option.

```(fields.E160) The options auto_now, auto_now_add, and default are mutually exclusive. Only one of these options may be present.```

but it is impossible to add field to existing model without adding default value and it is necessary to add value during `makemigrations` command - and there, as a default is `timezone.now`

Reply:
Yes, exactly, you have to remove the defaults. `auto_now_add` will handle it on its own.

Reply:
OK, done. Really strange :D